tasks/configure-gpg.yml

```diff
@@ -0,0 +1,159 @@
+---
+
+# check when GPG Space already exists
+
+- name: "Get GPG Spaces from API"
+  uri:
+    url: "{{ archivematica_src_configure_ss_url }}/api/v2/space/?access_protocol=GPG"
+    headers:
+      Authorization: "ApiKey {{ archivematica_src_configure_ss_user }}:{{ archivematica_src_configure_ss_api_key }}"
+    status_code: 200
+    method: GET
+  register: gpg_list_gpg_spaces
+
+# add GPG space and locations when there is no GPG space already configured
+- name: "Add GPG space and locations"
+  block:
+
+    - name: "Set gpg binary name"
+      set_fact:
+        gpg_binary: "gpg1"
+      when: "ansible_distribution_version | version_compare('18.04', '>=')"
+
+    - name: "Create batch file for GPG key"
+      template:
+        src: "gpg/gpg-batch-file.j2"
+        dest: "/tmp/gpg-batch-file"
```

Comment:
Is this file deleted after the process finishes?

Reply:
Good catch!!